packages/frontend/amp/components/AMPJSON.tsx

```diff
@@ -0,0 +1,12 @@
+// tslint:disable:react-no-dangerous-html
+import React from 'react';
+
+export const AMPJSON: React.FC<{ o: any }> = ({ o }) => {
```

Comment:
Naming pernicktetiness: 

- Why all caps? 
- Is this AMP-specific or is it just a way of injecting a JSON script tag into the document?

Reply:
I've ever seen a JSON script in AMP, but yes you're right, it might be more general. Do you know?

Reply:
I don't know. Given this is in the `amp` directory anyway, the `AMP` prefix is superfluous.  Maybe I'd just call it `JsonScript` or something?